CONFIGURATION.md

```diff
@@ -40,6 +40,7 @@ log_level                                |  *  | 0 .. 7          |             6
 log.queue                                |  *  | true, false     |         false | Disable spontaneous log_cb from internal librdkafka threads, instead enqueue log messages on queue set with `rd_kafka_set_log_queue()` and serve log callbacks or events through the standard poll APIs. **NOTE**: Log messages will linger in a temporary queue until the log queue has been set. <br>*Type: boolean*
 log.thread.name                          |  *  | true, false     |          true | Print internal thread name in log messages (useful for debugging librdkafka internals) <br>*Type: boolean*
 log.connection.close                     |  *  | true, false     |          true | Log broker disconnects. It might be useful to turn this off when interacting with 0.9 brokers with an aggressive `connection.max.idle.ms` value. <br>*Type: boolean*
+background_event_cb                      |  *  |                 |               | Background queue event callback (set with rd_kafka_conf_set_background_event_cb()) <br>*Type: pointer*
```

Comment:
it'd be good to beef this out a little bit. what are 'background queues'? what are the rules for using them?

Reply:
They're explained in the reference (set_background_event..), and this property is not settable with a standard string,string key-value API.
These C-type config properties (callbacks, opaques, etc) are included in CONFIGURATION.md as a reference, but they do have little value, we should probably take them all out at some point.